### ISSUE TITLE
Accuracy stuck at ~50%

### ISSUE THREAD

**Issue by risangbaskoro** [C23-PS419/petanikita-ml#9]
So when training, whatever the num of epoch is, the accuracy stucked at 50%. When testing on a test dataset, the accuracy only reaches 30%.

Some parameters that may need attention:
- Last layer activation (`sigmoid` or `softmax`?)
- Loss function
- Learning rate

It also may be beneficial to use callbacks such as LearningRateScheduler, ReduceLROnPlateau, etc.
